docs/release_notes/draft.md

```diff
@@ -0,0 +1,13 @@
+# Draft
+
+## Features
+
+- lowercased, user-focused description of the new feature (#0)
+
+## Improvements
```

Comment:
What is the difference between features and improvements?

Reply:
These terms predates me -- see other release notes -- so I cannot say for sure.
However, my understanding is that:
- a "feature" is a bigger (usually functional) change which typically adds capabilities to `eksctl`, e.g. support for GPU nodes, support for some new AWS feature, etc.
- an "improvement" would be a smaller (functional or non-functional, but usually non-functional) change which modifies `eksctl`'s behaviour, makes it operate more reliably, faster, with more transparency, etc.

Reply:
:+1: What marc said. Also improvements can be additions to docs.